jaeger-core/src/test/java/io/jaegertracing/senders/InMemorySender.java

```diff
@@ -34,33 +32,32 @@
   private Semaphore semaphore = new Semaphore(Integer.MAX_VALUE);
 
   public InMemorySender() {
```

Comment:
we used to have inMemoryReporter. What's the point of InMemorySender?

Reply:
The `InMemorySender` was in the `reporters` package. I moved it to this package, as it seems more appropriate.

Reply:
ok, just for tests - nvm